df.ui.xml

```diff
@@ -599,17 +599,25 @@
             <stl-vector name='stop_conditions' pointer-type='stop_depart_condition'/>
             <stl-vector name='stop_links' pointer-type='route_stockpile_link'/>
 -->
-            <int32_t/><int32_t/><int32_t/><int32_t/><int32_t/> 0.50.01
+            <int32_t/><int32_t/><int32_t/> 0.50.01
+            <bitfield name="flags">
+                <flag-bit name="children_do_chores" />
+            </bitfield>
+            <int32_t/> 0.50.01
             <stl-vector name='work_details' pointer-type='work_detail'/> 0.50.01
-            <int32_t/><int32_t/><int32_t/><int32_t/><int32_t/><int32_t/><int32_t/><int32_t/> 0.50.01
-            <int32_t/><int32_t/><int32_t/><int32_t/><int32_t/><int32_t/><int32_t/><int32_t/> 0.50.01
-            <int32_t/><int32_t/><int32_t/><int32_t/><int32_t/><int32_t/> 0.50.01
+            <static-array type-name='bool' name='chores' index-enum='unit_labor' count='94'/>
+
+            <stl-vector name='chores_exempted_children'> sorted
+                <int32_t ref-target='unit'/>
+            </stl-vector>
 
+<!--
```

Comment:
were these fields just wrong before? why are they commented out now? and can they just be deleted instead of commented?

Reply:
I did not check older versions, I guess they were wrong since 0.50.01.

They could be deleted. I followed the trend started by @quietust in cc7c6b149d9c0916aafa36702879700cd24aed68. I think quietust did that so it would be easier to recreate the field when they are found. But now we know there is not enough space for them to be in `hauling`. If they still exist, they are much further away.

Reply:
Indeed, the main reason I commented things out was because a _lot_ of things had changed so I didn't know at the time whether those fields had been removed or simply relocated.

Once it's clear that something has actually been removed, it should be erased from the XML - if it ever comes back, we can look back in the commit history to find it.

Reply:
I removed all the comments.